services/libfuzzer/setup.sh

```diff
@@ -53,7 +53,6 @@ packages=(
 package_recommends=(
   subversion
   ubuntu-restricted-addons
-  wget
```

Comment:
wget is not needed?

Reply:
I didn't think so but you made me double check. `oss-fuzz` does use wget for some reason.